blocks/columns/columns.css

```diff
@@ -286,10 +296,14 @@
 }
 
 @media (min-width: 768px) {
-  .columns-container[data-bg-image]{
+  .linear-gradient.columns-container[data-bg-image]{
     background-image: linear-gradient(to right, rgb(0 0 0 / 100%) 20%, rgb(0 0 0 / 0%)), var(--bg-image-url);
   }
 
+  .columns-container[data-bg-image]{
+    background-image: var(--bg-image-url);
+  }
+
   .columns.bg-image > div {
     width: calc(100% - 30%)!important;
```

Comment:
Could you also make this specific to the linear-gradient as it's only needed there?